### ISSUE TITLE
Comparing operators with HAVING queries

### ISSUE THREAD

**Issue by rrhyne** [yiisoft/yii2#13406]
### What steps will reproduce the problem?
the 'having' active record function does not support operators in the manner it did in yii1. This is analogous to the following issue which was addressed with the andFilterCompare function:

https://github.com/yiisoft/yii2/issues/2022

### What is the expected result?
That the having function would support a query such as:
$query->having("count", "> 100");

### What do you get instead?
The current function returns the sql:  count = '> 100' 

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.10
| PHP version      |
| Operating system |

**Comment by Alex-Code** [comment]
You can just specify it as a string.
```php
$query->having("count > 100");
```
or
```php
$query->having("count > :count", [":count" => 100]);
```

**Comment by klimov-paul** [comment]
You can specify 'having' condition with exact same syntax as 'where' condition.
Use:

```
$query->having(['>', 'count', '100']);
```

**Comment by rrhyne** [comment]
Thank you for your comments. The use case not supported by the suggestions here is gridview filters where the query would be created as so:

$query->having('quantityTotal', $this->quantityTotal);

This is one of those bits of magic that REALLY make Yii sing, and in my opinion should be standardized as it is with normal queries.

**Comment by klimov-paul** [comment]
https://github.com/yiisoft/yii2/blob/master/framework/db/Query.php#L883-L918

**Comment by rrhyne** [comment]
Ok, so it looks like that functionality will be released in 2.0.11? Thanks!

**Comment by cebe** [comment]
no, having() does that already longer, its only filterHaving() that is new.
